pom.xml

```diff
@@ -551,14 +549,17 @@
             </plugins>
         </pluginManagement>
         <plugins>
+            <!--
+                The flatten plugin is required to resolve the "revision" property used in the version tag of POMs in a
+                multi-module setup, see https://maven.apache.org/maven-ci-friendly.html#Install_.2F_Deploy.
+            -->
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>flatten-maven-plugin</artifactId>
                 <version>1.1.0</version>
```

Comment:
The most recent version of the flatten-maven-plugin is 1.2.1. It will probably make sense to upgrade the plugin.

Reply:
Good point, done.